src/main/webapp/i18n/en/guidedTourCourseExerciseOverview.json

```diff
@@ -36,20 +63,30 @@
             "startExercise": {
                 "headline": "Start with your first programming exercise",
                 "content": "Click on the 'Start exercise' button to start working on the exercise. This action takes a few seconds, since a personal repository is setup for you in the background."
-            }
-        },
-        "cloneRepository": {
-            "clone": {
-                "headline": "Clone repository",
-                "content": "Click on the 'Clone repository' button to clone the repository."
-            },
-            "useSourceTree": {
-                "headline": "Clone in SourceTree",
-                "content": "Click on the 'Clone in SourceTree' button to clone the repository in SourceTree. Alternatively, you can also copy the displayed URL and paste it into your git client."
-            },
-            "sourceTreeTutorial": {
-                "headline": "SourceTree Tutorial",
-                "content": "If you are not familiar with SourceTree, then have a look at our short tutorial on it."
+            },
+            "cloneRepository": {
+                "headline": "Clone repository in SourceTree",
+                "content": "Watch the following video to understand how to clone a git repository in SourceTree."
+            },
+            "inspectSourceTree": {
+                "headline": "Inspect SourceTree",
+                "content": "Watch the following video to understand the basic functions in SourceTree."
+            },
+            "importEclipse": {
+                "headline": "Import project in Eclipse",
+                "content": "Watch the following video to understand how to import the project in Eclipse."
+            },
+            "inspectProject": {
+                "headline": "Inspect Project",
+                "content": "In this video step we will inspect the project and solve the given task."
+            },
+            "commitAndPush": {
+                "headline": "Submit your changes",
+                "content": "Watch the following video to understand how to commit and push your changes."
+            },
+            "reviewResult": {
+                "headline": "Review your result",
+                "content": "After you have committed and pushed your changes as described in the previous video, the result will be automatically refreshed and displayed here. Artemis uses test cases to automatically assess your solution. If you now click on the exercise panel, you will get to the exercise detail page, review the test case results."
```

Comment:
Hm the last part `, review the test case results` was a bit frustrating for me. Did you miss the `to` word here?